test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -200,7 +200,10 @@ public void COMInterfaceWithSupportedOSPlatform(bool net50, bool allowMarshaling
             "ISpellCheckerFactory", // COM interface that includes `ref` parameters
             "LocalSystemTimeToLocalFileTime", // small step
             "ID3D12Resource", // COM interface with base types
-            "ID2D1RectangleGeometry")] // COM interface with base types
+            "ID2D1RectangleGeometry", // COM interface with base types
+            "GetCurrentProcess", // A handle for the current process
+            "GetCurrentProcessId", // An ID of the current process
+            "GetCurrentThreadId")] // An ID of the current thread of the current process
```

Comment:
I don't see the value of testing these individually unless they provide a faster path to exposing an existing test failure in the `FullGeneration` test.